docs/source/usage/usage.rst

```diff
@@ -212,6 +212,10 @@ Stories require the following fields to be filled out:
 	One or more images related to this Product.
 *Videos*
 	One or more videos related to this Product.
+*Add New Images*
+	New Images can be added by clicking on this button, a popup form for images will appear and desired information can be added and a new image is submitted.
```

Comment:
I would move this info into the Images and Videos section just above this (and the same for other sections like this) -- ``One or more images related to this Product. New images can be added by ...``